setup.py

```diff
@@ -33,11 +36,9 @@
         "Programming Language :: Python :: 3.9",
     ],
     install_requires=requirements,
-    setup_requires=setup_requirements,
-    tests_require=test_requirements,
-    name="pace-driver",
+    name="pace",
```

Comment:
Do we need to update more of this info here in the setup block here?

Reply:
I think this may be a good issue as I am not sure what would be appropriate to retain relating to the AI2 work.